### PR TITLE
Profile page sections setup

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Create profile page sections

## Testing done
Tested locally and QA reviewed

## Screenshots
image

## Acceptance criteria
- [x] Create dummy sections for each possible section
- [x] Each dummy section has correct CSS (i.e borders and header css) and spacing between sections
- [x] Each dummy section can be scrolled to by jump links in the currently built header
- [x] No conditional logic for hiding/showing section+jumplink is in this story
- [x] No content in each section, just headers with empty content area

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/gi-sandbox/components/IconWithInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function IconWithInfo({ icon, children, present }) {
+  if (!present) return null;
+  return (
+    <p className="icon-with-info">
+      <i className={`fa fa-${icon}`} />
```
[the remark this answers is not in the document]
Comment (timwright12) [reply src/applications/gi-sandbox/components/IconWithInfo.jsx:7]: @dmccurdybah can we add `aria-hidden="true"` to this icon?

File: src/applications/gi-sandbox/components/profile/JumpLink.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function JumpLink({ label, jumpToId }) {
+  return (
+    <a className="jump-link arrow-down-link" href={`#${jumpToId}`}>
+      <p>
+        <i className={`fa fa-arrow-down`} />
```
[the remark this answers is not in the document]
Comment (timwright12) [reply src/applications/gi-sandbox/components/profile/JumpLink.jsx:7]: `aria-hidden="true"` so screen readers don't announce the glyph

[the remark this answers is not in the document]
Comment (dmccurdybah) [reply src/applications/gi-sandbox/components/profile/JumpLink.jsx:7]: can do

**Review by timwright12 — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**